user_guide/src/examples/quickstart/__main__.py

```diff
@@ -0,0 +1,105 @@
+from ..paths import OUTPUT_BASE_DIR, create_if_not_exists
+from .series_tuple import series as s1
+from .series_list import series as s2
+from .dataframe1 import df as df1
+from .dataframe2 import df as df2
+from .dataframe3 import df as df3
+from .head import out as h1
+from .tail import out as t1
+from .sample import out as s1
+from .describe import out as d1
+from .select1 import out as sel1
+from .select2 import out as sel2
+from .select3 import out as sel3
+from .select4 import out as sel4
+from .filter1 import out as f1
+from .filter2 import out as f2
+from .withcolumns import out as wc1
+from .groupby1 import out as gb1
+from .groupby2 import out as gb2
+from .combine1 import out as c1
+from .combine2 import out as c2
+from .join_df1 import df1 as j1
+from .join_df2 import df2 as j2
+from .qs_join1 import out as join1
+from .concat1 import out as concat1
+
+path = create_if_not_exists(f"{OUTPUT_BASE_DIR}/quickstart")
+
```

Comment:
I've done these writes in a loop to reduce the amount of copying. See here for an example:https://github.com/pola-rs/polars-book/blob/master/user_guide/src/examples/combining_data/__main__.py

Reply:
Is the preferred way forward? We can rewrite it. Probably have to create a little guide for it, as I copied my approach from the current examples.

Reply:
@braaannigan what do you think? Merge it or rewrite it to your suggested method?